docs/content/docs/custom-resource/reference.md

```diff
@@ -78,8 +78,6 @@ This page serves as a full reference for FlinkDeployment custom resource definit
 
 | Value | Docs |
 | ----- | ---- |
-| v1_13 |  |
-| v1_14 |  |
```

Comment:
@gyfora Do the changes look good?

Reply:
Yes, I wonder maybe we could tweak the CRD reference generator to ignore `@Deprecated` fields , that would allow us to nudge users in the right direction :D

Reply:
Yeah, that would be nice. I looked into the doclet generator briefly, but actually adding a JavaDoc comment already allows us to inform users. See https://github.com/apache/flink-kubernetes-operator/pull/768/files?w=1#diff-283ef288346ac9001a197714b5307823f425eb89111554ecb7ab17f81ad9529fR81